Add meta description to some pages (#3853)
wazuh/wazuh-documentation: source/installation-guide/wazuh-agent/wazuh_agent_package_hpux.rst

.. Copyright (C) 2021 Wazuh, Inc.

.. meta::

>   :description: Learn more about how to successfully install the Wazuh agent on HP-UX systems in this section of our Installation guide.

Reviewer: ```suggestion
  :description: Learn more about how to successfully install the Wazuh agent on HP-UX systems in this section of our Installation Guide.
```

>   :description: Learn more about how to install or deploy the Wazuh lightweight agent in HP-UX in this section of our installation guide. 

Reviewer: ```suggestion
  :description: Learn more about how to successfully install the Wazuh agent on HP-UX systems in this section of our Installation guide.
```
